### DISCUSSION TITLE
Website not loading

### DISCUSSION THREAD

**Discussion by hshiferaw** [vercel/community#2284] [Help] [answered]
### Summary

my build works fie locally and remote and Vercel STATUS is Read, however [webpage](https://hshiferaw.me) is not loading, the issue started last night tried to update everything still same issue.

**Comment by amyegan** [comment]
Hi @hshiferaw! Do you see any error details in the [Runtime Logs](https://vercel.com/docs/concepts/observability/runtime-logs#)?

**Reply by hshiferaw** [reply]
Thank you for the help @amyegan 
<img width="1484" alt="Screenshot 2023-05-03 at 3 18 59 PM" src="https://user-images.githubusercontent.com/17464413/236022172-f7fcfd6c-8f50-4dbf-ab02-fb08bc97e26a.png">

**Reply by hshiferaw** [reply]
they all are “Info” level messages

**Reply by amyegan** [reply]
It looks like you found a solution! Is this still a problem for you?

**Reply by hshiferaw** [reply, marked as answer]
I used yarn package manager than npm, and for some reason, it fixed the issue; somewhat weird, but yea, that was the solution.
